modules/videopress/shortcode.php

```diff
@@ -129,6 +129,9 @@ function videopress_shortcode_callback( $attr ) {
  */
 wp_oembed_add_provider( '#^https?://videopress.com/v/.*#', 'http://public-api.wordpress.com/oembed/1.0/', true );
 
+// Register a VideoPress handler for direct links to .mov files (and potential other non-handled types later).
+wp_embed_register_handler( 'video', '#^https?://videos.(videopress.com|files.wordpress.com)/.+?\.(mov)$#i', apply_filters( 'wp_video_embed_handler', 'wp_embed_handler_video' ), 10 );
```

Comment:
Can we just add it to the line above? I left it like this, because I took the line right out of core and just changed the regex.